### PR TITLE
fix(curriculum): Added clarification for what text-align does

### LINKED ISSUE #55018: [Building a Cafe Menu] Clarify what `text-align` does in step 11 and 12
## Description

In step 11 and 12 of Building a Cafe Menu, we ask the users to use `text-align: center` to:

> Center your `h1` element

or

> Center the `h2` and `p` elements

But what the property does is centering the elements' _content_, rather than the elements themselves.

This is minor, but I think we should update the wording to be clear.

## Affected pages

- https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-11
- https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-12

## Suggested changes

Perhaps changing the instructions to:

```md
Center the content of your `h1` element...
```

And 

```md
Center the content of the `h2` and `p` elements...
```

### PR DESCRIPTION
**Pull request by sanjeevmurmu**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55018

### CODE REVIEW

**Review by a2937 — APPROVED** [review]
Looks Good To Me! 👍

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477ae9675db8bb7655b30.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-12
 
 # --description--
 
-In the previous step, you used a <dfn>type selector</dfn> to style the `h1` element. Center the `h2` and `p` elements by adding a new type selector for each one to the existing `style` element.
+In the previous step, you used a <dfn>type selector</dfn> to style the `h1` element. Center the content of `h2` and `p` elements by adding a new type selector for each one to the existing `style` element.
```
Comment (a2937) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477ae9675db8bb7655b30.md:10]: Wait I see a minor issue. You are missing a `the`.

**Review by a2937 — CHANGES_REQUESTED** [review]
Wait I see a minor issue.

**Review by sanjeevmurmu — COMMENTED** [review]
Fixed the grammatical error.

**Review by gikf — APPROVED** [review]
👍 
- - -
Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/#/how-to-contribute-to-the-codebase?id=contributing-to-the-codebase.

**Review by a2937 — APPROVED** [review]
Looks Good To Me! 👍

**Pull request merged**